gulpfile.babel.js

```diff
@@ -54,29 +53,36 @@ var webpackConfig = {
         test: /\.jsx|.js$/,
         exclude: /node_modules\//,
         loaders: [
-          'babel-loader?stage=1&plugins[]=object-assign'
-        ]
+          'babel-loader?stage=1&plugins[]=object-assign',
+        ],
       },
     ],
     postLoaders: [
       {
-        loader: "transform/cacheable?brfs"
-      }
-    ]
+        loader: "transform/cacheable?brfs",
+      },
+    ],
   },
   resolve: {
-    extensions: ['', '.js', '.jsx']
+    extensions: ['', '.js', '.jsx'],
   },
-  plugins: gulpPlugins
+  plugins: gulpPlugins,
 };
 
-gulp.task('test-unit', function (done) {
+gulp.task('lint', function() {
+  return gulp.src([paths.gulpfile, paths.src, paths.example])
```

Comment:
I wonder if we should just have it lint all `js(x)` by default, and add `dist` and `node_modules` to `.eslintignore`? It would be nice to not have to update these paths.

I think this would mean reformatting `karma.conf.js` to pass linting, too. Thoughts?